templates/apollo-starter-ts/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "apollo-rest-template",
-  "version": "0.8.3",
+  "version": "0.8.4",
```

Comment:
@wtrocki I wasn't sure if I am to add this manually or is this step done (semi) automatically?